apps/nowcasting-app/components/helpers/utils.ts

```diff
@@ -573,3 +573,30 @@ export const createBucketObject: (
     increment
   };
 };
+export function calculateChartYMax(
+  chartData: Array<{ [key: string]: number | string | number[] }> | null | undefined,
```

Comment:
Please set this `chartData` param to the `ChartData` type so we can enforce the type properly 👍